File: DevExpress-Demo/CS/DiagramDemo.Wpf/ViewModels/DatabaseDiagramViewModel.cs
using DevExpress.Diagram.Demos;
using DevExpress.Mvvm;
using DevExpress.Xpf.Diagram;

namespace DiagramDemo {
    public class DatabaseDiagramViewModel : ViewModelBase {
        public DatabaseDefinition Database { get; private set; }

        public DatabaseDiagramViewModel() {
            Database = DatabaseData.GetDatabaseDefinition();
        }
    }
}

File: DevExpress-Demo/VB/DiagramDemo.Wpf/ViewModels/DatabaseDiagramViewModel.vb
Imports DevExpress.Diagram.Demos
Imports DevExpress.Mvvm

Namespace DiagramDemo

    Public Class DatabaseDiagramViewModel
        Inherits ViewModelBase

        Private _Database As DatabaseDefinition

        Public Property Database As DatabaseDefinition
            Get
                Return _Database
            End Get

            Private Set(ByVal value As DatabaseDefinition)
                _Database = value
            End Set
        End Property

        Public Sub New()
            Database = GetDatabaseDefinition()
        End Sub
    End Class
End Namespace
